_i18n/en.yml

```diff
@@ -35,7 +35,7 @@ index:
   column_1:
     heading: Simple and secure for the public
     content:
-      p_1: Users need fewer passwords and learn fewer interfaces. Security experts protect one service instead of many. Dedicated teams of design and security experts will continuously improve it.
+      p_1: Because login.gov is a shared service, users need fewer passwords and learn fewer interfaces. Also, security experts protect one service instead of many. And dedicated teams of design and security experts will continuously improve it.
```

Comment:
@esgoodman If you don't like beginning w/ an "and" then include "also" in the last sentence, or omit it altogether.